roles/irods_icommands/templates/irods_icommands.pref.j2

```diff
@@ -0,0 +1,7 @@
+# {{ ansible_managed }}
+
+{% set irods_server = irods_icommands_package_new.split('=') %}
+Explantion: Pin added by role: irods_icat
```

Comment:
role is `irods_icommands` (only for this file)

And `Explantion` -> `Explanation` (also for other files)